posthog/models/plugin.py

```diff
@@ -552,6 +552,10 @@ def validate_plugin_job_payload(plugin: Plugin, job_type: str, payload: dict[str
 
 @receiver(models.signals.post_save, sender=Organization)
 def preinstall_plugins_for_new_organization(sender, instance: Organization, created: bool, **kwargs):
+    # Skip plugin installation if HOG transformations are enabled
+    if getattr(settings, "HOG_TRANSFORMATIONS_ENABLED", True):
```

Comment:
Sorry to be a pain here but we need to think this out.

We will likely turn on hog transformations first _before_ making this the default as it won't be an instant rollout. So we probably need a specific setting for this action.

Also the setting doesnt exist.